Automattic/wp-calypso
# Welcome Guide: Remove podcasting-specific guide

#### Changes proposed in this Pull Request

* Revert the changes made in https://github.com/Automattic/wp-calypso/pull/48357 since we're no longer showing a podcasting-specific welcome guide

#### Testing instructions



* Create a new user and podcasting site and check that the normal welcome tour displays without errors
* Create a new user and regular site and check that the normal welcome tour displays without errors



Related to 411-gh-Automattic/dotcom-manage
